app/src/main/java/com/openvehicles/OVMS/ui2/pages/EnergyFragment.kt

```diff
@@ -127,41 +130,86 @@ class EnergyFragment : BaseFragment(), OnResultCommandListener, EnergyMetricsAda
         val battVolt = findViewById(R.id.battVolt) as TextView
         val battAmp = findViewById(R.id.battAmp) as TextView
         val battkW = findViewById(R.id.battkW) as TextView
-        battVolt.text = String.format("%2.1f V", carData?.car_battery_voltage)
-        battAmp.text = String.format("%2.1f A", carData?.car_battery_current_raw)
-        battkW.text = String.format("%2.2f kW", carData?.car_power)
+        battVolt.text = String.format("%2.0f V", carData?.car_battery_voltage)
```

Comment:
%.1f precision on the battery voltage is important for low voltage vehicles like the Twizy. If you need the space, apply %.1f only to voltages < 100.